content/management/configuration-api/v2.0/index.mdx

```diff
@@ -300,6 +312,10 @@ curl "https://api.livechatinc.com/agents/john.doe@mycompany.com" \
 
 Creates a new agent in your license.
 
+### Required scopes
+
+- `agents_write`
```

Comment:
We always require `accounts--all:rc`, and additionally for PPS we require `agents--all:rw`